### PR TITLE
Source JUPITER_SWAP_URL from env

### PR DESCRIPTION
**Pull request by theelderbeever**
Allows for the Jupiter swap url to be configured via an environment variable while defaulting to the original hardcoded value.

### CODE REVIEW

**Comment by abunsen** [comment]
Can someone review?

File: rust/switchboard-utils/src/task/jupiter_swap.rs [hunk 1]
```diff
@@ -98,11 +98,12 @@ impl JupiterSwapClient {
...
     ) -> String {
         let mut url = format!(
-            "https://quote-api.jup.ag/v6/quote?inputMint={}&outputMint={}&amount={}&slippageBps={}&onlyDirectRoutes=false", 
+            "{base_url}/quote?inputMint={}&outputMint={}&amount={}&slippageBps={}&onlyDirectRoutes=false",
             in_token_address,
             out_token_address,
             amount,
-            slippage_bps.unwrap_or(100.0)
+            slippage_bps.unwrap_or(100.0),
+            base_url = std::env::var("JUPITER_SWAP_URL").unwrap_or("https://public.jupiterapi.com/".to_string())
```
Comment (abunsen) [inline rust/switchboard-utils/src/task/jupiter_swap.rs:106]:
```suggestion
            base_url = std::env::var("JUPITER_SWAP_URL").unwrap_or("https://quote-api.jup.ag/v6/quote".to_string())
```

File: rust/switchboard-utils/README.md [hunk 2]
```diff
@@ -34,3 +34,8 @@ Or add the following line to your Cargo.toml:
 [dependencies]
 switchboard-utils = "0.8.0"
 ```
+
+## Environment Variables
+
+- `JUPITER_SWAP_URL` - default: https://public.jupiterapi.com/
```
Comment (abunsen) [inline rust/switchboard-utils/README.md:40]:
```suggestion
- `JUPITER_SWAP_URL` - for higher rate limits use: https://public.jupiterapi.com/
```

**Review by abunsen — CHANGES_REQUESTED** [review]

File: rust/switchboard-utils/src/task/jupiter_swap.rs (hunk 1 above)
Comment (theelderbeever) [reply rust/switchboard-utils/src/task/jupiter_swap.rs]: Changed to `api.jup.ag/v6` since `/quote` is in the formatted string already.

**Review by mgild — APPROVED** [review]

**Pull request merged**